frontend/model/database.js

```diff
@@ -388,6 +403,27 @@ sbp('sbp/selectors/register', {
   },
   'gi.db/filesCache/clear': async function (): Promise<void> {
     await filesCache.clear()
+  },
+  'gi.db/filesCache/temporary/save': function (cacheKey: string, blob: Blob): Promise<*> {
+    if (cacheKey.startsWith('__')) throw new Error('Invalid key')
+    return sbp('gi.db/filesCache/save', `temporary/${cacheKey}/`, blob)
+  },
+  'gi.db/filesCache/temporary/load': function (cacheKey: string): Promise<Blob> {
+    if (cacheKey.startsWith('__')) throw new Error('Invalid key')
+    return sbp('gi.db/filesCache/load', `temporary/${cacheKey}/`)
+  },
+  'gi.db/filesCache/temporary/delete': function (cacheKey: string): Promise<void> {
+    if (cacheKey.startsWith('__')) throw new Error('Invalid key')
+    return sbp('gi.db/filesCache/delete', `temporary/${cacheKey}/`)
+  },
+  'gi.db/filesCache/temporary/clear': function () {
+    sbp('okTurtles.eventQueue/queueEvent', 'gi.db/files', async () => {
+      const keys = await filesCache.getItem('keys') ?? []
+      const allTempKeys = keys.filter(k => k.startsWith('temporary/'))
+      await filesCache.removeMany(allTempKeys).catch(e => {
+        console.error('[gi.db/filesCache/temporary/clear] Error removing temporary keys', e)
+      })
+    })
```

Comment:
For the context, these were created in an attempt to implement this request of Greg:

> So we can't use sessionStorage for this. I would remove that code and instead use the 'gi.db/filesCache/* namespace for this, while keeping track of what files are saved so that all of them can be cleared when the site first loads.

So the key for a blob that we want to store temporarily is prefixed with `temporary/` which will be cleared when the app loaded via `sbp('gi.db/filesCache/temporary/clear)`. Hope it looks good.